src/pages/index.tsx

```diff
@@ -1,49 +1,50 @@
-import { Component, Element, State, h } from '@stencil/core';
+import React from 'react';
```

Comment:
this is the homepage. I think it's something like 90% converted over to render ~the same as on the existing site.